runtime/v2/model/src/test/resources/schema/concord.yml

```diff
@@ -47,8 +47,8 @@ flows:
     - form: "formName"
     - throw: ${error}
     - call: "default2"
-      in:
-        k: "v"
+#      in:
+#        k: "v"
```

Comment:
Is this intended be be left commented out?

Reply:
it's commented because `ConcordJsonSchemaGeneratorTest` failed. do not know why, maybe bug in json validator. do not want to waste time on this, so just commented this lines :)